lightpack.py

```diff
@@ -494,12 +554,14 @@ def fail(cause = None):
 			fail(e)
 
 		# Check greeting and reported API version
-		match = re.match(r'^Lightpack API v(\S+)', greeting)
-		version = StrictVersion(match.group(1))
-		if version < API_VERSION_GTE or version > API_VERSION_LTE:
-			fail("API version (%s) is not supported" % version)
-		if not match:
-			print(match)
+		match = re.findall(r'API v?(\d+(?:\.\d+)?)', greeting)
```

Comment:
I don't like that the "start of string" anchor was removed here. Was that necessary?

Reply:
The Prismatik fork appends its version after Lightpacks' original string, like so: `Lightpack API v1.4 - Prismatik API v2.2 (type "help" for more info)`. This updated regex is indeed looser on the position, but to compensate is stricter on the version matching.

Reply:
OK.